rails/Gemfile

```diff
@@ -8,6 +8,11 @@ module ::Kernel
   def mysql2_prepared_statements?
     ENV["MYSQL2_PREPARED_STATEMENTS"] == '1'
   end
+  
+  def v_4_0_x?
+    major, minor, micro = ENV["RAILS_VERSION"].to_s.split(".")
```

Comment:
It would be awesome if I could use `Rails.version` instead of `ENV["RAILS_VERSION"]` but it is not recognized here.

@noahgibbs Do you can think of any way to require Rails specified in Gemfile in order to use it in this function?

Reply:
Oh hey, that makes sense :-( Nope, that would be hard to do. You wouldn't want to depend on the Rails version being well known because this has to work even before the Rails gem is installed... You *could* make the Rails version be an in-Gemfile variable and use it both to specify the version of the Rails gem above and to check v_4_0_x?. But I don't know if that would be worth the trouble.

Reply:
I'll leave it as it is right now then -- since we are passing `RAILS_VERSION` environment variable to [container](https://github.com/ruby-bench/ruby-bench-docker/blob/master/scripts/rails/releases.sh#L30) running suite.

Reply:
Sounds good to me.